package.json

```diff
@@ -8,11 +8,11 @@
     "test": "test"
   },
   "scripts": {
-    "test": "npm run build:ts && tsc -p test/tsconfig.json && tap --ts \"test/**/*.test.ts\"",
-    "start": "npm run build:ts && fastify start -l info dist/app.js",
+    "test": "bun install && tsc -p test/tsconfig.json && tap --ts \"test/**/*.test.ts\"",
```

Comment:
We don't need these redundant `bun install` directives with every command.

Reply:
By directives you mean the command itself, or the commands that are following the `bun install` command?

Reply:
The bun install commands preceding the actual command